CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,120 @@
+# Code of Conduct
+
+We follow the [CNCF Code of Conduct][cncf-coc].
```

Comment:
Why re-invent yet another CoC? The CNCF CoC is designed for general use in all CNCF community. If the CNCF CoC is not able to serve the purpose here, please state the requirement and use case and bring the discussion to the TOC. It is way beyond the scope of this WG to re-scope the CNCF CoC.

Reply:
> Why re-invent yet another CoC? 

Not really. It looks like this PR introduced the TAG Security CoC to the working group. 

IMO: let's move on quickly.... 
Option1: either merge this PR, which is fine from my point of view since we are reusing the TAG-Security structure, or. 
Option2: remove this file and refer to the CNCF CoC in a single line only.

I currently prefer option2. (I don't think the additional CoC stuff is necessary and could be confusing).

Reply:
+1 
Let's leverage on the CNCF tooling and Keep the draft lean and fast forward to action.

Reply:
This review was already given, therefore it would be great to have this comment on the corresponding feedback.
However, we don't reinvent the CoC. As the first line states: We follow the CNCF CoC. 
How much more we should use the CNCF CoC?

The first part explains how we want to follow this commitment. The second part clarifies incident handling which the CoC doesn't because every project has to manage this by itself. (The reporting part of the CoC is like this - someone uses for example a religious emoji, because it looks like K8s, in the wrong context and rather as an adult person just solve this but highlight your concerns, you go to the highest escalation point known. Think this approach is not sustainable, and therefore mentioned encouraging people to bring up their points like you would in other cases.)

The next part is about references, which are not managed in CoC.

If this is such a problem we can move it somewhere else into own files or whatsoever.

Reply:
> This review was already given, therefore it would be great to have this comment on the corresponding feedback. However, we don't reinvent the CoC. As the first line states: We follow the CNCF CoC. How much more we should use the CNCF CoC?
> 
If CNCF CoC is not able to cover the use case, voice the requirement and get the feedback before re-scoping it.

> The first part explains how we want to follow this commitment. The second part clarifies incident handling which the CoC doesn't because every project has to manage this by itself. (The reporting part of the CoC is like this - someone uses for example a religious emoji, because it looks like K8s, in the wrong context and rather as an adult person just solve this but highlight your concerns, you go to the highest escalation point known. Think this approach is not sustainable, and therefore mentioned encouraging people to bring up their points like you would in other cases.)
> 
Per CNCF CoC, there are two layers of escalation: project maintainer and CNCF. Any evidence why this process is not sustainable?
> The next part is about references, which are not managed in CoC.
Again, if this is not something covered by CNCF CoC, open an issue and have it resolved, it is beneficial to the community.
> 
> If this is such a problem we can move it somewhere else into own files or whatsoever.

Reply:
This is still outstanding as an issue.

Reply:
@catblade the file contains only a reference to the CNCF CoC (no more information)